script-test/playbackstrategies/msestrategytest.js

```diff
@@ -1,491 +1,579 @@
+            setUpMSE(0, WindowTypes.GROWING, MediaKinds.VIDEO, 100000, 200000);
 
-      describe('tearDown()', function () {
-        it('should reset the MediaPlayer', function () {
-          setUpMSE();
-          mseStrategy.load('src', null, 0);
+            mockDashInstance.getSource.and.returnValue('src');
 
-          mseStrategy.tearDown();
+            mseStrategy.load('src', null, undefined);
 
-          expect(mockDashInstance.reset).toHaveBeenCalled();
-        });
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src', true);
+          });
 
-        it('should tear down bindings to MediaPlayer Events correctly', function () {
-          setUpMSE();
-          mseStrategy.load(null, null, 0);
-
-          mseStrategy.tearDown();
-
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('timeupdate', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('playing', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('pause', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('waiting', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('seeking', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('seeked', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('ended', jasmine.any(Function));
-          expect(mockVideoElement.removeEventListener).toHaveBeenCalledWith('error', jasmine.any(Function));
-          expect(mockDashInstance.off).toHaveBeenCalledWith(dashjsMediaPlayerEvents.ERROR, jasmine.any(Function));
-          expect(mockDashInstance.off).toHaveBeenCalledWith(dashjsMediaPlayerEvents.QUALITY_CHANGE_RENDERED, jasmine.any(Function));
-          expect(mockDashInstance.off).toHaveBeenCalledWith(dashjsMediaPlayerEvents.METRIC_ADDED, jasmine.any(Function));
-          expect(mockDashInstance.off).toHaveBeenCalledWith(dashjsMediaPlayerEvents.MANIFEST_LOADED, jasmine.any(Function));
-          expect(mockDashInstance.off).toHaveBeenCalledWith(dashjsMediaPlayerEvents.MANIFEST_VALIDITY_CHANGED, jasmine.any(Function));
-        });
+          it('returns the correct duration from the DASH Mediaplayer with audio', function () {
+            setUpMSE(null, null, MediaKinds.AUDIO);
 
-        it('should remove the video element', function () {
-          setUpMSE();
-          mseStrategy.load(null, null, 0);
+            mseStrategy.load('src', null, 0);
 
-          expect(videoContainer.childElementCount).toBe(1);
+            expect(mseStrategy.getDuration()).toBe(101);
+          });
 
-          mseStrategy.tearDown();
+          it('returns the correct duration from the DASH Mediaplayer with video', function () {
```

Comment:
This seems to be a duplicate of `'returns the correct duration from the DASH Mediaplayer'` further down in the `'getDuration()'` describe block (which is also a more appropriate describe block for this test).